client/object_put.go

```diff
@@ -222,6 +232,11 @@ func (x *DefaultObjectWriter) Close() error {
 
 	defer x.cancelCtxStream()
 
+	if x.streamClosed {
+		err = x.err
```

Comment:
BTW, this can lead to counting the error twice, which isn't what we want. Likely no statistics should be collected at all (and context cancelling can be done in `Write()` itself.

Reply:
@smallhive, statistics is not smth i understand what for, so can you, please answer whether logical errors should be passed to that handler?

Reply:
or... i finally got you. anyway, recheck, please